src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements CalculateArea, Printer {
```

Comment:
```suggestion
public class Circle extends Figure {
```
let's implement this interfaces in the Figure class